core/src/main/java/com/csi4999/screens/SavedEntitiesScreen.java

```diff
@@ -26,13 +30,22 @@ public class SavedEntitiesScreen implements Screen {
     private Stage stage;
 
     // set to true if no DB implemented
-    private final Boolean noDatabase = true;
+    private final Boolean noDatabase = false;
```

Comment:
is there a reason the Boolean object is used instead of the boolean primitive?

Reply:
no I just always forget Java has boolean as a privative for some reason ... not really sure why
anyway I fixed it